netroute_bsd_test.go

```diff
@@ -0,0 +1,74 @@
+package netroute
```

Comment:
this test is getting picked up on other platforms.

probably needs a
`//go:build darwin || dragonfly || freebsd || netbsd || openbsd` line?